Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -149,13 +151,29 @@ public extension VisualInstruction.Component {
     }
 }
 
+public struct GuidanceViewImageRepresentation: Equatable {
```

Comment:
ImageRepresentation isn’t meant to be limited to the `image` case; it just means that it contains the details for rendering an image. So does this struct, but the difference is in how the URL is interpreted. I wonder if we should try to use ImageRepresentation for both cases but genericize it to make sense in both. If we must have a separate struct, we should probably rename the two structs to better reflect the distinction. Perhaps one would be ScalableImageRepresentation or something to that effect. Needs more thought.

/ref https://github.com/mapbox/mapbox-directions-swift/pull/405#discussion_r365487164